### PR TITLE
Updating links following AWS-B migration

### PR DESCRIPTION
**Pull request by franmoore05**
### What

Updated prod links following AWS-B migration.

### How to review

Ensure the changes make sense and the links work.

### Who can review

Anyone with prod access.

### CODE REVIEW

**Review by mr-nick17 — DISMISSED** [review]
LGTM

File: training/platform-services/PLATFORM.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ More information about Nomad can be found on their website - https://www.nomadpr
 ## Vault
 Visit the following to access Vault
 - DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://vault.develop.onsdigital.co.uk
-- PRODUCTION:&nbsp;&nbsp;&nbsp;https://vault.production.onsdigital.co.uk
+- PROD:&nbsp;&nbsp;&nbsp;https://vault.dp-prod.aws.onsdigital.uk
```
[the remark this answers is not in the document]
Comment (franmoore05) [reply training/platform-services/PLATFORM.md]: Good spot @rafahop - I've now updated it

[the remark this answers is not in the document]
Comment (rafahop) [reply training/platform-services/PLATFORM.md]: Sorry @franmoore05 but there are a couple of sections missing: Nomad and Consul

[the remark this answers is not in the document]
Comment (franmoore05) [reply training/platform-services/PLATFORM.md]: These should be fixed now.

**Review by rafahop — APPROVED** [review]

**Pull request merged**